src/Giveaway.js

```diff
@@ -590,6 +600,63 @@ class Giveaway extends EventEmitter {
             }
         });
     }
+
+    /**
+     * Pause's the giveaway
+     * @param {GiveawayPauseOptions} options The pause options
+     * @returns {Promise<Giveaway>} The paused giveaway
+     */
+     pause(options = {}) {
+        return new Promise(async (resolve, reject) => {
+            if (this.ended) {
+                return reject('Giveaway with message ID ' + this.messageID + ' is already ended.');
+            }
+            if (!this.channel) {
+                return reject('Unable to get the channel of the giveaway with message ID ' + this.messageID + '.');
+            }
+            await this.fetchMessage().catch(() => {});
+            if (!this.message) {
+                return reject('Unable to fetch message with ID ' + this.messageID + '.');
+            }
+            if(this.pauseOptions.isPaused) return reject('Giveaway with ID ' + this.messageID + ' is already paused.');
+            // Update data
+            if (options.content && typeof options.content === "string") this.pauseOptions.content = this.options.pauseOptions.content;
+            if (options.unPauseAfter && !isNaN(options.unPauseAfter)) this.pauseOptions.unPauseAfter = Date.now() + this.options.pauseOptions.unPauseAfter;
+            if (options.embedColor && typeof options.embedColor === "string") this.pauseOptions.embedColor = this.options.pauseOptions.embedColor;
+            this.pauseOptions.isPaused = true;
+            const embed = this.manager.generatePauseEmbed(this);
+            this.message.edit({ embed }).catch(() => {});
+            // Call the db method
+            await this.manager.editGiveaway(this.messageID, this.data);
+            resolve(this);
+        });
+    }
+
+    /**
+     * UnPause's the paused giveaway
+     * @returns {Promise<Giveaway>} The unpaused giveaway
+     */
+     unpause() {
+        return new Promise(async (resolve, reject) => {
+            if (this.ended) {
+                return reject('Giveaway with message ID ' + this.messageID + ' is already ended.');
+            }
+            if (!this.channel) {
+                return reject('Unable to get the channel of the giveaway with message ID ' + this.messageID + '.');
+            }
+            await this.fetchMessage().catch(() => {});
+            if (!this.message) {
+                return reject('Unable to fetch message with ID ' + this.messageID + '.');
+            }
+            if(!this.pauseOptions.isPaused) return reject('Giveaway with ID ' + this.messageID + ' is not paused.');
```

Comment:
```suggestion
            if (!this.pauseOptions.isPaused) {
                 return reject('Giveaway with ID ' + this.messageID + ' is not paused.');
             }
```

Reply:
I will maybe ask Androz somewhen if we can change to single line format cause I personally think it saves space and looks better when are able to do it in 1 line.
But for now I guess we should stick to the existing format.